x-pack/test/functional/apps/observability_logs_explorer/config.ts

```diff
@@ -37,9 +43,31 @@ export default async function createTestConfig({
   const functionalConfig = await readConfigFile(require.resolve('../../config.base.js'));
   const services = functionalConfig.get('services');
 
+  const packageRegistryConfig = path.join(__dirname, './common/package_registry_config.yml');
+  const dockerArgs: string[] = ['-v', `${packageRegistryConfig}:/package-registry/config.yml`];
+
+  /**
+   * This is used by CI to set the docker registry port
+   * you can also define this environment variable locally when running tests which
+   * will spin up a local docker package registry locally for you
+   * if this is defined it takes precedence over the `packageRegistryOverride` variable
+   */
+  const dockerRegistryPort: string | undefined = process.env.FLEET_PACKAGE_REGISTRY_PORT;
+
   return {
     ...functionalConfig.getAll(),
     testFiles: [require.resolve('.')],
+    dockerServers: defineDockerServersConfig({
+      registry: {
+        enabled: !!dockerRegistryPort,
+        image: dockerImage,
+        portInContainer: 8080,
+        port: dockerRegistryPort,
+        args: dockerArgs,
+        waitForLogLine: 'package manifests loaded',
+        waitForLogLineTimeoutMs: 60 * 2 * 10000, // 2 minutes
```

Comment:
> [!WARNING]
> `60 * 2 * 10000` is going to be 20 minutes timeout, I think there is an extra `0` in `10000`

Reply:
Good catch! Addressed in [c9c06](https://github.com/elastic/kibana/pull/192064/commits/c9c06037f294e72fffce8990a398fd1586f912f1).